plugins/module_utils/azure_rm_common.py

```diff
@@ -1252,6 +1256,19 @@ def lock_models(self):
         self.log("Getting lock models")
         return ManagementLockClient.models('2016-09-01')
 
+    @property
+    def datalake_store_client(self):
+        self.log('Getting datalake store client...')
+        if not self._datalake_store_client:
+            self._datalake_store_client = self.get_mgmt_svc_client(DataLakeStoreAccountManagementClient,
```

Comment:
1264 and 1265 lines continuation line under-indented for visual indent

Reply:
Done, thx!